src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -253,7 +295,12 @@ if (routeData) {
 }
 ```
 
-##### `app.match(request)`
+##### `app.match()`
+
+<p>
+
+**Type:** `(request: Request) => RouteData | undefined`
```

Comment:
You can [check the type here](https://github.com/withastro/astro/blob/a2f8c5d85ff15803f5cedf9148cd70ffc138ddef/packages/astro/src/core/app/index.ts#L164C27-L164C48).